portal.go

```diff
@@ -1042,17 +1078,32 @@ func (portal *Portal) handleIMMemberChange(msg *imessage.Message, dbMessage *dat
 	return nil
 }
 
-func (portal *Portal) handleIMAttachment(msg *imessage.Message, attach *imessage.Attachment, intent *appservice.IntentAPI) (*event.MessageEventContent, error) {
+func (portal *Portal) handleIMAttachment(msg *imessage.Message, attach *imessage.Attachment, intent *appservice.IntentAPI) (*event.MessageEventContent, map[string]interface{}, error) {
 	data, err := attach.Read()
 	if err != nil {
 		portal.log.Errorfln("Failed to read attachment in %s: %v", msg.GUID, err)
-		return nil, fmt.Errorf("failed to read attachment: %w", err)
+		return nil, nil, fmt.Errorf("failed to read attachment: %w", err)
+	}
+
+	mimeType := attach.GetMimeType()
+	fileName := attach.GetFileName()
+	extraContent := map[string]interface{}{}
+	if msg.IsAudioMessage {
+		data, err = ffmpeg.ConvertBytes(data, ".ogg", []string{}, []string{"-c:a", "libopus"}, ".caf")
```

Comment:
Shouldn't the last parameter be a mime type?

Reply:
https://github.com/mautrix/go/pull/61

Reply:
It was working with this incorrectly set so :shrug: